Domain.Sql/DatabaseExtensions.cs

```diff
@@ -135,14 +136,15 @@ public static bool IsAzureDatabase(this IDbConnection connection)
             this DbContext context, 
             int dbSizeInGB = 2, 
             string edition = "standard", 
-            string serviceObjective = "S0")
+            string serviceObjective = "S0",
+            string connectionString = null)
```

Comment:
This is confusing. Why would we want a connecting string other than the one configured for the `DbContext`?